lib/chef/knife/bootstrap_windows_ssh.rb

```diff
@@ -60,14 +59,20 @@ class BootstrapWindowsSsh < Bootstrap
         :description => "The ssh gateway",
         :proc => Proc.new { |key| Chef::Config[:knife][:ssh_gateway] = key }
 
+      option :forward_agent,
+        :short => "-A",
+        :long => "--forward-agent",
+        :description => "Enable SSH agent forwarding",
+        :boolean => true
+
       option :identity_file,
         :short => "-i IDENTITY_FILE",
         :long => "--identity-file IDENTITY_FILE",
         :description => "The SSH identity file used for authentication"
 
-      option :host_key_verification,
-        :long => "--[no-]host-key-verification",
-        :description => "Disable host key verification",
+      option :host_key_verify,
```

Comment:
If we change this we probably need some kind of backwards-compat / deprecation message?